host/frontend/host-orchestrator/operator.go

```diff
@@ -45,6 +45,7 @@ func startHttpServer() {
 }
 
 func startHttpsServer() {
+	tlsCertDir := fromEnvOrDefault("ORCHESTRATOR_TLS_CERT_DIR", defaultTLSCertDir)
```

Comment:
please add a commented out example for this configuration to debian/cuttlefish-common.default

Reply:
done